pyTigerGraph/pyTigerGraph.py

```diff
@@ -22,12 +23,13 @@ class TigerGraphConnection(pyTigerGraphVertex, pyTigerGraphEdge, pyTigerGraphUDT
     pyTigerGraphLoading, pyTigerGraphPath):
     """Python wrapper for TigerGraph's REST++ and GSQL APIs"""
 
-    def __init__(self, host: str = "http://127.0.0.1", graphname: str = "MyGraph",
+    def __init__(self, host: str = "http://34.232.63.121", graphname: str = "BankSim",
```

Comment:
We don't want to change this default. The new address is only relevant to us but not to a general user.

Reply:
I changed it